pbsmetrics/go_metrics_test.go

```diff
@@ -1,14 +1,18 @@
 package pbsmetrics
 
 import (
+	"net/http/httptrace"
 	"testing"
+	"time"
 
 	"github.com/prebid/prebid-server/config"
 	"github.com/prebid/prebid-server/openrtb_ext"
 	metrics "github.com/rcrowley/go-metrics"
 	"github.com/stretchr/testify/assert"
 )
 
+const OneSecond time.Duration = time.Second
```

Comment:
`time.Second` is already a constant, any reason why we wanna define it as a constant again over here with a diff name?

Reply:
Good point. Corrected